test/test_pipeline_baseline.py

```diff
@@ -184,13 +184,29 @@ def test_go_gapic_client_baseline(tmpdir):
                       tmpdir)
 
 
-def _test_java_baseline(task_name, test_name, tmpdir):
-    _test_baseline(task_name, test_name, 'java', str(tmpdir), {})
+def _test_java_baseline(task_name, test_name, tmpdir, extra_kwargs={}):
```

Comment:
Guess I missed this in the Python review as well -- please fix that test, too. (Separately from this PR is OK.)